manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/BridgeServiceImpl.java

```diff
@@ -231,54 +237,59 @@ public Bridge updateBridgeStatus(ResourceStatusDTO statusDTO) {
                     statusDTO.getGeneration());
             return bridge;
         }
+
         Operation operation = bridge.getOperation();
         List<Condition> conditions = bridge.getConditions();
+        // Set the updated conditions to the existing Manager conditions to begin; then copy in the new Operator conditions
         List<Condition> updatedConditions = conditions.stream().filter(c -> c.getComponent() == ComponentType.MANAGER).collect(Collectors.toList());
         statusDTO.getConditions().forEach(c -> updatedConditions.add(Condition.from(c, ComponentType.SHARD)));
+
+        // Don't do anything if Conditions are unchanged from our previous state.
+        ManagedResourceStatusV2 status = StatusUtilities.getManagedResourceStatus(operation, conditions);
+        ManagedResourceStatusV2 originalStatus = StatusUtilities.getManagedResourceStatus(operation, updatedConditions);
+        if (Objects.equals(status, originalStatus)) {
+            LOGGER.info("Update for Bridge with id '{}' was discarded. The ManagedResourceStatus reflected by the Conditions is unchanged.", bridge.getId());
+            return bridge;
+        }
+
```

Comment:
I see the point, but I'm wondering if this is conceptually wrong. Example: 
1) we deploy the bridge `mybridge` with the version `A` of the operator
2) it gets deployed properly by the operator, the manager gets updated, all good.
3) After some months we update the operator  with the version `B`, and we add some new conditions/whatever.
4) The manager gets the update with the new set of conditions, but the manager discards them because the resource is already `ready`

As well as we might discard updates on the `lastTransitionTime` if the resource gets updated but it does not change the status.

Reply:
As per chat, we agreed to add a column in the DB to persist when we completed the user's request as per diagram 
![metrics drawio](https://user-images.githubusercontent.com/18282531/212364571-3154cf9e-6ecc-473a-b2fa-0479c540ff0f.png)